puppeteer/bundleRequests.test.js

```diff
@@ -70,7 +70,7 @@ describe('Js bundle requests', () => {
                 .forEach(url => {
                   expect(url).toMatch(
                     new RegExp(
-                      `(\\/static\\/js\\/(?:comscore\\/)?(main|framework|commons|shared|${serviceRegex}|.+Page)-.+?.js)|(\\/static\\/.+?-lib-.+?.js)`,
+                      `(\\/static\\/js\\/(?:comscore\\/)?(main|framework|commons|shared|${serviceRegex}|.+Page).+?.js)|(\\/static\\/.+?-lib.+?.js)`,
```

Comment:
What triggered this change?

Reply:
see https://github.com/bbc/simorgh/pull/9255/files#r673958016 for before and after bundle names